### PR TITLE
Sending case management emails

### PR DESCRIPTION
**Pull request by peterdavidhamilton**
## Changes in this PR

- enable the "send email to school" action in case management
- convert the class to use the template UUID now that a single service is used in all environments
- update the tests accordingly
- convert templates to follow the same convention for variables as used in support requests
  + an object that has methods `first_name`, `last_name` and `email` 

## Next steps

- Ensure template content is correct and links to the prototype have been changed.
- Identify why some emails arrived with duplicate greetings and signatures??

### CODE REVIEW

**Comment by peterdavidhamilton** [comment]
[Sending emails to school](https://trello.com/c/0JIUWIZv/338-sending-emails-to-school)

File: app/services/support/emails/to_school.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require "notify/email"
+
+module Support
+  class Emails::ToSchool < Notify::Email
+    def call
+      Rollbar.info "Sending email to school"
```
Comment (d-a-v-e) [inline app/services/support/emails/to_school.rb:8]: Does this need any more context passed to Rollbar? is eventually for a slack notification?

Comment (peterdavidhamilton) [reply app/services/support/emails/to_school.rb:8]: Perhaps but we shouldn't identify the user. We will see this in the slack channel, so for now, it is just letting us know the attempt was made.

**Review by AndrewCowle — APPROVED** [review]

File: app/controllers/support/cases/emails_controller.rb [hunk 2]
```diff
@@ -4,27 +4,39 @@ def create
       @case_email_content_form = CaseEmailContentForm.from_validation(validation)
 
       if validation.success?
-        create_email_interaction(@case_email_content_form.email_body)
+
```
[the remark this answers is not in the document]
Comment (peterdavidhamilton) [reply app/controllers/support/cases/emails_controller.rb:7]: Don't think you have looked at the latest code @AndrewCowle

File: app/controllers/support/cases/emails_controller.rb [hunk 3]
```diff
@@ -4,27 +4,39 @@ def create
...
+      params.require(:case_email_content_form).permit(:email_body, :email_subject, :email_template)
+    end
+
+    def send_email_to_school
+      Support::Emails::ToSchool.new(
+        recipient: @current_case,
+        template: params.fetch(:email_template, "ac679471-8bb9-4364-a534-e87f585c46f3"),
+        reference: @current_case.ref,
+        variables: {
+          text: @case_email_content_form.email_body,
```
Comment (ryantk) [inline app/controllers/support/cases/emails_controller.rb:36]: We probably want to have the same variables here as in preview
https://github.com/DFE-Digital/buy-for-your-school/blob/766fc83a49ee43b080f0dc59352fc60a05265f51/app/controllers/support/cases/emails/content_controller.rb#L53

**Review by ryantk — APPROVED** [review]

**Pull request merged**